charts/seed-bootstrap/dashboards/controller-details.json

```diff
@@ -1272,12 +1349,51 @@
       {
         "allValue": null,
         "current": {
+          "isNone": true,
           "selected": false,
-          "text": "health-progressing",
-          "value": "health-progressing"
+          "text": "None",
+          "value": ""
         },
         "datasource": "seed-prometheus",
-        "definition": "label_values(controller_runtime_reconcile_total{job=\"$job\"}, controller)",
+        "definition": "label_values(controller_runtime_reconcile_total{job=\"$job\"},app_kubernetes_io_instance)",
```

Comment:
Instead of adding a new grafana variable, can we overwrite the `job` label, please?
I got the impression, that it's a well-known practice in prometheus to keep the `job` label distinct for individual targets.
The `name` variable is confusing because its empty for `gardenlet` and `resource-manager`.
Also, it's easy to forget to select the `app_kubernetes_io_instance` in a panel query. It would be more maintainable if developers would only need to select the job label.

You could add something like this to the `extensions` and `garden` scrape config:
```yaml
      - source_labels: [ __meta_kubernetes_pod_label_app_kubernetes_io_instance ]
        target_label: job
```

You could even consider combining both the `extensions` and `garden` scrape config.

Reply:
Addressed in https://github.com/gardener/gardener/commit/d6d468a3e02f9d4cd78cb357cb74815342940003